src/server/game/Server/WorldSocket.cpp

```diff
@@ -385,7 +385,10 @@ WorldSocket::ReadDataHandlerResult WorldSocket::ReadDataHandler()
     }
 
     // Our Idle timer will reset on any non PING opcodes on login screen, allowing us to catch people idling.
-    _worldSession->ResetTimeOutTime(false);
+    if (packetToQueue->GetOpcode() != CMSG_WARDEN_DATA)
```

Comment:
is this not better set within ResetTimeOutTime()?

Reply:
to catch every instance where we call ResetTimeOutTime

Reply:
?? I don't understand.

Reply:
dont worry, I was wrong anyways!